webextension/js/popup.js

```diff
@@ -85,6 +98,17 @@ const Logic = {
     });
   },
 
+  clearBrowserActionBadge() {
+    getExtensionInfo().then(extensionInfo=>{
+      browser.storage.local.get({browserActionBadgesClicked: []}).then(storage=>{
+        browser.browserAction.setBadgeBackgroundColor({color: ""});
+        browser.browserAction.setBadgeText({text: ""});
+        storage.browserActionBadgesClicked.push(extensionInfo.version);
+        browser.storage.local.set({browserActionBadgesClicked: storage.browserActionBadgesClicked});
+      }).catch(e=>{throw e;});
+    }).catch(e=>{throw e;});
```

Comment:
Huh, I though I tried that and it failed to parse the `async`, but it works now. Cleaned up.